scenarios/nomad-consul-vault-secure/README.md

```diff
@@ -0,0 +1,134 @@
+# Scenario: Nomad Consul Vault Quickstart
+This scenario deploys Nomad, Consul and Vault with security configurations in place. This scenario is useful when you have to play around the features of Nomad and Vault (with or without Consul ) with the security aspects and runs workload Identity for Vault.
+
+## Prerequisites
+The following tools are required to run these scenarios:
+
+- Shikari and Lima with Socket_VMNet configured
+- Requires a base VM image built using packer (../../packer/hashibox.pkr.hcl)
+- Uses qemu driver (you can use vz by modifying vmType in the template)
+- The vault config uses raft as storage.
```

Comment:
We don't have to duplicate these requirements in every scenario anymore, as they are already available in the repo's main README.